components/innerHeader.tsx

```diff
@@ -2,20 +2,23 @@ import { Box, Button, H1, HR, Text } from '@bigcommerce/big-design';
 import { ArrowBackIcon } from '@bigcommerce/big-design-icons';
 import { useRouter } from 'next/router';
 import { useProductList } from '../lib/hooks';
-import { TabIds, TabRoutes } from './header';
+import { RouteTitles, TabRoutes } from './header';
 
 const InnerHeader = () => {
     const router = useRouter();
     const { pid } = router.query;
     const { list = [] } = useProductList();
     const { name } = list.find(item => item.id === Number(pid)) ?? {};
 
-    const handleBackClick = () => router.push(TabRoutes[TabIds.PRODUCTS]);
+    const routeHead = router.pathname.split('/')[1];
+    const title = RouteTitles[routeHead];
```

Comment:
Given that this is an `innerHeader`, and homepage will never be inner, we could probably drop `Home` from `RouteTitles`. Also, since we're not actually consuming `RouteTitles` inside of the main header page, we should probably move that const to this file.

```jsx
export const RouteTitles = {
    PRODUCTS: 'Products',
    ORDERS: 'Orders',
};
```